service_openrc_linux.go

```diff
@@ -166,15 +166,14 @@ func (s *openrc) Run() (err error) {
 }
 
 func (s *openrc) Status() (Status, error) {
-	_, out, err := runWithOutput("rc-service", s.Name, "status")
-	if err != nil {
-		return StatusUnknown, err
-	}
+	_, out, _ := runWithOutput("rc-service", s.Name, "status") // a stopped service will return an error here, so we're not checking the error code
```

Comment:
Modify this comment so:
 1. It comes before.
 2. It documents return codes (if exists and started=exit 0, stopped=exit 3, not exist=exit 1
 3. If you continue to use error text, please also check explicitly for "does not exist" suffix, then any other match should return actual output if there is no match with something like `return StatusUnknown, fmt.Errorf("Unknown OpenRC status messge: %s", out)`

Reply:
I feel like we should inspect the error type, to ensure that it ran, but exited with non-zero, as well. Then if for some reason rc-service doesn't exist or something else strange, it reports it.

Reply:
addressed in the latest commit.